web_src/less/helpers.less

```diff
@@ -6,6 +6,7 @@
 .ac { align-items: center !important; }
 .tc { text-align: center !important; }
 .tl { text-align: left !important; }
+.tdn { text-decoration: none !important; }
```

Comment:
should this be notdn instead?

Reply:
Hmm, so far these helpers were all abbreviations. GitHub uses `no-underline` for the same rule, could also use that.

Reply:
ah it stands for **t**ext **d**irection **n**one not  **t*ext **d**irectio**n**

Reply:
**t**ext-**d**ecoration: **n**one 😉